src/app/groups/containers/group-header/group-header.component.html

```diff
@@ -1,7 +1,37 @@
-<div class="group-header-container" *ngIf="groupData">
+<div class="group-header-container">
   <div class="group-header">
     <div class="group-title-container">
-      <h1 class="group-title">{{ groupData.group.name }}</h1>
+      <h1 class="group-title alg-h1">{{ groupData().group.name }}</h1>
+      @if (state$ | async; as state) {
+        <p class="group-subtitle">
+          <ng-container i18n>Associated activity: </ng-container>
+            @if (state.isFetching) {
+              <span i18n>loading...</span>
+            } @else if (state.isError) {
+              @if ($any(state.error).status === 403) {
```

Comment:
We'll have to find a clean way to address these cases in the future.

Reply:
I was thinking about pipe approach, as you done for permissions

Reply:
ah good idea!